run.sh

```
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Check if yq is installed
+if ! command -v yq &> /dev/null
+then
+    echo "yq could not be found, please install it first"
+    exit 1
+fi
+
+# Extract dictionary IDs from URLs
+DICT_IDS=($(yq e '.rime.urls | to_entries | .[].value | split("/") | .[-1]' config.yml))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)